exodus/static/css/exodus.css

```diff
@@ -22,6 +22,19 @@ li.nav-item.active > a {
 nav.navbar {
   font-size: 1rem !important;
 }
+h1.main-title {
+  color: #684971;
+  font-weight: bold;
+  margin-bottom: 15px;
+}
+a.main-link {
+  color: #684971;
+  font-weight:bold;
+}
+a.main-link:after {
```

Comment:
I've tried to center the `>` with the text and this magic thing `vertical-align: -10%;` looks to do the trick.

Reply:
I'll try that, thanks!